docs/ACCEPTANCE_TESTS.md

```diff
@@ -0,0 +1,97 @@
+# Rule acceptance tests 
+
+## Goal
+Execute the validator against all datasets (latest version hosted in the MobilityDatabase) and quantify the effect of a code change on all of them.  
+The source of truth is defined as the latest stable version of the validator (the on available on [the master branch of this repository](https://github.com/MobilityData/gtfs-validator/tree/master)). 
+
+For the latest version of all GTFS datasets from the MobilityArchives, the validation report from both the snapshot and the source of truth are compared. An acceptance test report is generated: it quantifies for each agency/dataset the number of new errors (as defined [here](https://github.com/MobilityData/gtfs-validator/blob/master/RULES.md#definitions)) in introduced.
```

Comment:
I would define "snapshot" and "source of truth" either in a bulleted list or in a narrative. I don't think it's clear what these are from the current text. What is "acceptance criteria" and acceptance mean?

I'd also relate this back to the "why" earlier - "snapshot" is the pull request with proposed changes, and we want to test this against the existing community-accepted set of rules (i.e., "source of truth") to make sure we're not breaking things for existing producers and consumers of GTFS around the world.

Reply:
I see "reference.json" is used to describe the report later - I think I prefer "reference" to "source of truth". I'm not a huge fan of "snapshot" either (especially because typically in Maven/Gradle language that refers to the latest unreleased version of the main branch), but I'm not sure I have good suggestions for alternatives - maybe "proposed"?